docs/config.json

```diff
@@ -110,7 +114,8 @@
           "entries": [
             {
               "title": "Scaling",
-              "slug": "/setup/operations/scaling/"
+              "slug": "/setup/operations/scaling/",
+              "hideInScopes": "cloud"
```

Comment:
It might be useful to have a separate scaling page for cloud customers.

If a cloud customer wants to connect thousands of nodes, do we just automatically detect the need to scale and do it for them, or is there a process by which they must request more capacity?

I don't know the answer to these questions, which suggests that customers probably don't either, and a docs page would be helpful.

Reply:
That makes sense—I've added an issue here: https://github.com/gravitational/teleport/issues/12836